src/dashboard/tests/test_rights.py

```diff
@@ -0,0 +1,336 @@
+# from components.rights.load import load_rights
+#
+# for premis_rights in fsentry.get_premis_rights():
+#     load_rights(premis_rights)
+#
+
+from metsrw.plugins import premisrw
+import pytest
+import uuid
+
+from components.rights import load
+from main.models import File, Transfer, SIP, MetadataAppliesToType
+
+
+_RIGHTS_STATEMENT_IDENTIFIER = (
```

Comment:
This is all personal preference, but I don't know if we need to worry about "private" constants in test — I think it should be obvious to everybody not to import from a test file.